lib/openstudio-aws.rb

```diff
@@ -16,13 +16,16 @@
 rescue LoadError
   puts 'Failed to load AWS-SDK-CORE gem'
   puts '  try running: gem install aws-sdk-core'
-  exit
+
+  exit 1
 end
 
+require 'openstudio/core_ext/hash'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.